src/app/qgsvectorlayerproperties.cpp

```diff
@@ -86,6 +88,15 @@ QgsVectorLayerProperties::QgsVectorLayerProperties(
   : QgsOptionsDialogBase( QStringLiteral( "VectorLayerProperties" ), parent, fl )
   , mLayer( lyr )
   , mOriginalSubsetSQL( lyr->subsetString() )
+  , mSaveAsMenu( nullptr )
+  , mLoadStyleMenu( nullptr )
+  , mRendererDialog( nullptr )
+  , labelingDialog( nullptr )
+  , mActionDialog( nullptr )
+  , diagramPropertiesDialog( nullptr )
+  , mFieldsPropertiesDialog( nullptr )
+  , mSourceFieldsPropertiesDialog( nullptr )
+  , mAttributesFormPropertiesDialog( nullptr )
```

Comment:
@m-kuhn could this initialisation be moved to the .h? just /me curious

Reply:
Yep

    s/could/should